Cargo.lock

```diff
@@ -541,7 +541,9 @@ dependencies = [
  "inventory 0.1.3 (registry+https://github.com/rust-lang/crates.io-index)",
  "ron 0.4.2 (registry+https://github.com/rust-lang/crates.io-index)",
  "serde 1.0.97 (registry+https://github.com/rust-lang/crates.io-index)",
+ "serde-importable-derive 0.1.0",
  "type-uuid 0.1.1 (registry+https://github.com/rust-lang/crates.io-index)",
+ "typetag 0.1.3 (registry+https://github.com/rust-lang/crates.io-index)",
  "uuid 0.7.4 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
Why is the Cargo.lock file in the git repo?

Reply:
No idea, do you want me to add it to gitigbore?

Reply:
I never worked in this repo, but usually the .lock files are in the gitignore ^^

Reply:
I believe the new guidelines are to leave it on the repo. Why it is like this however is unknown to me.

Reply:
Yeah the guidelines do state so for binaries: https://doc.rust-lang.org/cargo/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries

But atelier-assets is not primarily something that should be distributed independently, so I think it's fine to add it to the gitignore @Areredify !